src/verificationclassifier.py

```diff
@@ -162,6 +160,44 @@ async def do_command(self):
     async def get_object_point_clouds(self):
         return
 
+    async def get_properties(
+            self,
+            *,
+            extra: Optional[Mapping[str, Any]] = None,
+            timeout: Optional[float] = None) -> Vision.Properties:
+        return Vision.Properties(
+                classifications_supported=True,
+                detections_supported=False,
+                object_point_clouds_supported=False,
+        )
+
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+        result = CaptureAllResult()
+        if camera_name != self.camera_name:
```

Comment:
I don't think we can do anything about this but I still dislike (even more now) that we need to include camera names in the config.